Makefile

```
@@ -53,7 +53,15 @@ docs: $(DECENTRALAND_ECS_TYPEDEF_FILE) ## Generate `decentraland-ecs` documentat
 	@cd $(PWD)/packages/decentraland-ecs; $(PWD)/node_modules/.bin/api-documenter markdown -i types/dcl -o docs
 	@echo "\nAPI Documentation was generated in the packages/decentraland-ecs/docs folder"
 
-build-essentials: $(COMPILED_SUPPORT_JS_FILES) $(DECENTRALAND_ECS_TYPEDEF_FILE) $(BUILD_ECS) $(SCENE_SYSTEM) $(INTERNAL_SCENES) $(DECENTRALAND_LOADER) generate-mocks ## Build the basic required files for the explorer
+RENDERER_BUILD := static/unity/Build/unity.data.unityweb static/unity/Build/unity.wasm.framework.unityweb static/unity/Build/unity.wasm.code.unityweb
+
+static/unity/Build/%.unityweb:
```

We need this to wire the Unity files in the `static` dir with the `node_modules`.
```suggestion
static/unity/Build/%.unityweb: node_modules/decentraland-renderer/%.unityweb
```